### PR TITLE
Storage server optional arguments

### PR DESCRIPTION
**Pull request by BeaudanBrown**
Added ability to include optional storage server command line arguments in launcher.ini

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,7 +111,20 @@ function launcherStorageServer(config, cb) {
     console.log('not going to start storageServer, shutting down')
     return
   }
-  storageServer = spawn(config.binary_path, [config.ip, config.port])
+  let optionals = []
+  if (config.log_level) {
+    optionals.push('--log-level', config.log_level)
+  }
+  if (config.lokinet_identity) {
```
Comment (neuroscr) [inline index.js:118]: is this the .loki, .snode or ?? address?

File: index.js [hunk 2]
```diff
@@ -111,7 +111,20 @@ function launcherStorageServer(config, cb) {
...
+    optionals.push('--lokinet-identity', config.lokinet_identity)
+  }
+  if (config.output_log) {
+    optionals.push('--output-log', config.output_log)
```
Comment (neuroscr) [inline index.js:122]: the launcher will eventually handle the logging, so we may make this an undocumentation option for release

File: index.js [hunk 3]
```diff
@@ -111,7 +111,20 @@ function launcherStorageServer(config, cb) {
...
+  }
+  if (config.db_location) {
+    optionals.push('--db-location', config.db_location)
```
Comment (neuroscr) [inline index.js:125]: what's the default if it's not passed in?

File: launcher.ini [hunk 4]
```diff
@@ -17,5 +17,10 @@ rpc_port      = 28083
 public_port   = 1090
 
 [storage]
-binary_path = src/loki-storage-server/build/httpserver
-port        = 8080
+binary_path      = src/loki-storage-server/build/httpserver
+ip               = 127.0.0.1
```
Comment (neuroscr) [inline launcher.ini:21]: this is removed now

Comment (BeaudanBrown) [reply launcher.ini:21]: Ah whoops, messed up rebase

File: index.js (hunk 3 above)
Comment (BeaudanBrown) [reply index.js:125]: Default is "."

File: launcher.ini (hunk 4 above)
Comment (neuroscr) [reply launcher.ini:21]: I fixed it, no worries

File: index.js (hunk 1 above)
Comment (BeaudanBrown) [reply index.js:118]: This is used for the storage server to get the keys used for encrypting the channels with messenger clients, and also is used to calculate the snode address of the server
Open to other ideas, lokid rpc endpoint or something?

**Pull request merged**